### PR TITLE
Publish solutions for lab 3

### PR DESCRIPTION
**Pull request by andreia-oca**

### CODE REVIEW

File: labs/lab03/skel/lin/1-system/my_system.c [hunk 1]
```diff
@@ -1,11 +1,11 @@
 /**
-  * SO, 2017
-  * Lab #3
-  *
-  * Task #1, lin
-  *
-  * Use of "system" library call and of "strace"
-  */
+ * SO
```
Comment (sketch873) [inline labs/lab03/skel/lin/1-system/my_system.c]: Trailing space

File: labs/lab03/skel/lin/2-orphan/orphan.c [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
-  * Lab #3
-  *
-  * Task #2, lin
-  *
-  * What happens if a parent process does not wait for its child process?
-  */
+ * SO
+ * Lab #3
+ *
+ * Task #2, lin
```
Comment (sketch873) [inline labs/lab03/skel/lin/2-orphan/orphan.c]: Linux

File: labs/lab03/sol/lin/3-tiny/tiny.c [hunk 3]
```diff
@@ -0,0 +1,228 @@
+/**
+ * SO
+ * Lab #3
+ *
+ * Task #3, Linux
```
Comment (sketch873) [inline labs/lab03/sol/lin/3-tiny/tiny.c]: Trailing space

File: labs/lab03/sol/lin/3-tiny/tiny.c [hunk 4]
```diff
@@ -0,0 +1,228 @@
...
+static char *expand(const char *key)
+{
+	/**
+	 * TODO - Return the value of environment variable
+	 */
+	return getenv(key);
+}
+
+/**
+ *  @args - array that contains a simple command with parrameters
```
Comment (sketch873) [inline labs/lab03/sol/lin/3-tiny/tiny.c]: Typo `parrameters`

File: labs/lab03/skel/lin/6-pipe/pipe.c [hunk 5]
```diff
@@ -84,7 +83,8 @@ int main(void)
 
 	/* TODO - Create anonymous pipe */
 
-	/* Sometimes I wish I could do a fork() in real life. No more
+	/**
+	 * Sometimes I wish I could do a fork() in real life. No more
 	 * hard choices to think about. Endless possibillites to explore. MH.
```
Comment (sketch873) [inline labs/lab03/skel/lin/6-pipe/pipe.c]: Typo `possibillites`

File: labs/lab03/sol/lin/6-pipe/pipe.c [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+	pid_t pid;
+	int rc;
+	int fds[2];
+
+	rc = pipe(fds);
+	DIE(rc != 0, "pipe() failed");
+
+	/**
+	 * Sometimes I wish I could do a fork() in real life. No more
+	 * hard choices to think about. Endless possibillites to explore. MH.
```
Comment (sketch873) [inline labs/lab03/sol/lin/6-pipe/pipe.c]: Typo `possibillites`

**Review by sketch873 — COMMENTED** [review]
Besides the inline comments please remove the year and the word `Procese` from the linux utils header.

**Review by sketch873 — APPROVED** [review]

**Review by lucigrigo — APPROVED** [review]
Looks good to me.

**Pull request merged**